packages/axelar-soroban-std/src/testutils.rs

```diff
@@ -71,6 +72,15 @@ where
     assert_emitted_event(env, -1, contract_id, topics, data);
 }
 
+// Converts Stellar address (soroban_sdk::Address) to soroban_sdk::Bytes
+pub fn address_to_bytes(env: &Env, address: &Address) -> Bytes {
```

Comment:
See `address.rs`. Add a [to_bytes](https://github.com/axelarnetwork/axelar-cgp-soroban/blob/8239e4126cdccb4156f737dd6e20fad5c2bfc239/packages/axelar-soroban-std/src/address.rs#L9) trait method to use this in a cleaner way